components/src/core/components/Input/password-input.scss

```diff
@@ -0,0 +1,123 @@
+.password-container {
+  position: relative;
+  width: 100%;
+  .password-strength-check {
+    position: absolute;
+    right: 0;
+    top: 0;
+    border-radius: 0.65rem;
+    margin-top: -25px;
+    font-family: Nunito;
```

Comment:
These value should be inherited from `oxd-text` component. this maybe redundant

Reply:
remove "font-family: Nunito;"
but if I remove other changes strength component is not visible properly